src/local_scheduler/local_scheduler.cc

```diff
@@ -594,25 +564,60 @@ void assign_task_to_worker(LocalSchedulerState *state,
   } else {
     Task_free(task);
   }
+
+  // If the task was an actor creation task, then do some extra bookkeeping.
+  if (TaskSpec_is_actor_creation_task(spec)) {
+    ActorID actor_creation_id = TaskSpec_actor_creation_id(spec);
+
+    ActorMapEntry entry;
+    entry.local_scheduler_id = get_db_client_id(state->db);
+    entry.driver_id = driver_id;
+    state->actor_mapping[actor_creation_id] = entry;
```

Comment:
Does it make sense to let the `handle_actor_creation_callback` handle adding the entry to the actor mapping? I guess it will add some extra latency though.

It would be nice to think about how to make the writes to `actor_mapping` idempotent, especially for handling reconstruction and suppression. I noticed that when you handle notifications from the table, you count a duplicate as an error, but when we receive our own notification, we'll have the actor in the table already. Not necessary for this PR, but just documenting that here.

Reply:
Hmm.. so the code path would be

1. actor creation task finishes
2. `finish_task` calls `handle_convert_worker_to_actor` and `publish_actor_creation_notification`
3. `handle_actor_creation_callback` adds the `ActorMapEntry`

Haven't thought through all the details but that might be cleaner since we are only adding the `ActorMapEntry` once. The latency in this case is probably not an issue.

Reply:
I'll give this a try

Reply:
fixed